### PR TITLE
update trap version

### PR DESCRIPTION
**Pull request by An0nJon**

### CODE REVIEW

**Comment by scglenn** [comment]
Lets hold off on merging until we have the functionality merged into the main dro repo and have dev docs updated

**Review by scglenn — CHANGES_REQUESTED** [review]

File: bridges/nomad/packages/interfaces/ITrap.sol
Comment (An0nJon) [reply bridges/nomad/packages/interfaces/ITrap.sol:6]: For the isValid params?

Comment (scglenn) [reply bridges/nomad/packages/interfaces/ITrap.sol:6]: isValid should be pure for ITrap.sol instead of view

**Review by scglenn — CHANGES_REQUESTED** [review]

**Review by scglenn — APPROVED** [review]

**Pull request merged**
